site2/docs/deploy-bare-metal.md

```diff
@@ -134,14 +133,13 @@ pulsar-io-aerospike-{{pulsar:version}}.nar
 ...
 ```
 
-## Installing Tiered Storage Offloaders (optional)
+## Install Tiered Storage Offloaders (optional)
```

Comment:
This part is the same as https://pulsar.apache.org/docs/en/next/standalone/#install-tiered-storage-offloaders-optional, so you can add a link here.